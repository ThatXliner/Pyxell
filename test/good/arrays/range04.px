
print [1..5 step 2]
print ['x'...'t' step -1]