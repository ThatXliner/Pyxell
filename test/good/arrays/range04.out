[1, 3, 5]
[x, w, v, u]
